interpreter/cling/tools/packaging/cpt.py

```diff
@@ -97,10 +97,12 @@ def travis_fold_end(tag):
 
 
 def box_draw_header():
-    msg = 'cling (' + platform.machine() + ')' + formatdate(time.time(), tzinfo())
+    msg = 'cling (' + platform.machine() + ')'
+    + formatdate(time.time(), tzinfo())
```

Comment:
I doubt that breaking the line there works (syntax error in line 101).  Perhaps a missing `\`? (same below)

Reply:
Yeah you're right I'll fix that thank you